documents/overview.md

```diff
@@ -0,0 +1,42 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>OID4VC demo overview 
+  <br>
+</h1>
+
+---
+
+**Warning: This project still is in very early development. Breaking changes without notice will happen at this point!**
+
+## Overview
+This multimodule project, consists of three parts. At the core of this project sits the "agent" module. This module coordinates all the backend functionalities of two other react projects. The agent can achieve this by taking advantage of our other ssi related technologies. Here are a brief list of other useful resources:
+- [SSI-SDK](https://github.com/Sphereon-Opensource/SSI-SDK)
+  - This mono repository, contains packages that add support for Presentation Exchange and OpenID4VC (SIOPv2, OID4VCI, OID4VP) and other functionalities to SSI-SDK and Veramo based agents.
+- [PEX](https://github.com/Sphereon-Opensource/PEX)
+  - The Presentation Exchange (PEX) Library implements the functionality described in the DIF Presentation Exchange specification for both version 1 and 2.
+- [OID4VCI](https://github.com/Sphereon-Opensource/OID4VCI)
+  - This is a mono-repository with a client and issuer pacakge to request and receive Verifiable Credentials using the OpenID for Verifiable Credential Issuance ( OpenID4VCI) specification for receiving Verifiable Credentials as a holder/subject.
+- [SIOP-OID4VP](https://github.com/Sphereon-Opensource/SIOP-OID4VP)
+  - An OpenID authentication library conforming to the Self Issued OpenID Provider v2 (SIOPv2) and OpenID for Verifiable Presentations (OpenID4VP) as specified in the OpenID Connect working group.
+
+And many more open-source libraries.
+You can use this demo for showcasing VCI abilities as well as SIOP-OID4VP abilities.
+
+In this document we're going to show you step by step what you have to do to set your own VCI demo up.
+Note that in order for you to set a VCI demo up and actually receive a credential, you need to have an SSI compatible wallet. Here is a list of wallet's with such capabilities:
+//TODO: complete this list
+- Sphereon
```

Comment:
Link this to the Sphereon Wallet Github and the Animo wallet Github for now

Reply:
linked (this is moved to project's readme)